src/mmg3d/mmg3d1.c

```diff
@@ -2593,6 +2631,7 @@ MMG3D_anatets_iso(MMG5_pMesh mesh,MMG5_pSol met,int8_t typchk) {
               fprintf(stderr,"\n  ## Warning: %s: surfacic pattern: unable to find"
                       " a valid split for at least 1 point. Point(s) deletion.\n",
                       __func__ );
+              MMG5_show_tet_location(mesh, pt, k);
```

Comment:
Please, keep this in debug mode only (`-d` command line option, `mesh->info.ddebug` variable inside the code) : 
```
if ( mesh->info.ddebug ) {
   MMG5_show_tet_location(mesh, pt, k);
}
```